manifests/init.pp

```diff
@@ -130,6 +130,12 @@
 #   available, likely results in performance degradation due to I/O blocking and is not recommended in most cases. Modifying this parameter should
 #   be done incrementally with benchmarking at each step to determine an optimal value for your deployment.
 #
+# @param service_enable
+#   Enable/disable Pulp services at boot.
+#
+# @param service_ensure
+#   Specify if Pulp services should be running.
```

Comment:
This makes it clear that the service will actually be stopped (and isn't unmanaged on `false`)
```suggestion
#   Specify if Pulp services should be running or stopped.
```